src/main/java/com/example/onlinebookstore/dto/BookRequestDto.java

```diff
@@ -1,19 +1,31 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
+    @Size(min = 2)
     private String title;
 
+    @NotNull
+    @Size(min = 2)
     private String author;
 
+    @NotNull
+    @Min(value = 0, message = "can't be less than 0")
     private BigDecimal price;
 
+    @NotNull
+    @Pattern(regexp = "\\d{13}", message = "ISBN must be a 13-digit number")
```

Comment:
There's annotation `@ISBN`, so you can use it with `@NotNull` to validate ISBN.
Just a suggestion 🙂